vm/call_frame.go

```diff
@@ -128,7 +128,9 @@ func (b *baseFrame) getLCL(index, depth int) (p *Pointer) {
 
 	if depth == 0 {
 		b.RLock()
-		p = b.locals[index]
+		if index < len(b.locals) {
```

Comment:
Or I think we can change this to
```go
if index >= len(b.locals) {
  return
}
p = b.locals[index]
```

so we can explicitly tell others that we intend to return nil in such situation. The original way looks like `p = b.locals[index]` is a special condition because it's put inside an if statement

Reply:
Cool, thanks. Will do.